recipes/oarfish/build.sh

```
@@ -2,5 +2,22 @@
 
+unamestr=`uname`
+if [ "$unamestr" == 'Darwin' ];
+then
+  export CC=clang
+  export CXX=clang++
+else
+  export CC=gcc
+  export CXX=g++
+fi
+
+alias cc=$CC
```

Shellcheck

[warning] This expands when defined, not when used. Consider escaping.

(SC2139)